src/test/store/markers.test.js

```diff
@@ -330,10 +333,21 @@ describe('Marker schema filtering', function() {
       ['payload no schema', 0, null, { type: 'no schema marker' }],
       ['RefreshDriverTick', 0, null, { type: 'Text', name: 'RefreshDriverTick' }],
       ['UserTiming',        5, 6,    { type: 'UserTiming', name: 'name', entryType: 'mark' }],
+      // The following is a tracing marker without a schema attached, this was a
+      // regression reported in Bug 1678698.
+      ['CompositeToTarget', 7, 8,    { type: 'tracing', category: 'Paint' }],
```

Comment:
Ah dang, you caught this before I could fix it :)

I saw it right before going to lunch, since the codecov wasn't hitting there. I'll fix it by using a dummy one, since other tests are relying on the existing behavior.